models/EventFeedbackModel.ts

```diff
@@ -0,0 +1,35 @@
+import { BaseEntity, Column, Entity, Generated, Index, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { PublicEventFeedback, Uuid } from '../types';
+import { EventModel } from './EventModel';
+import { UserModel } from './UserModel';
+
+@Entity('EventFeedback')
+export class EventFeedbackModel extends BaseEntity {
+  @Column({ select: false })
+  @Generated('increment')
+  id: number;
+
+  @PrimaryGeneratedColumn('uuid')
+  uuid: Uuid;
+
+  @ManyToOne((type) => EventModel, (event) => event.feedback, { nullable: false })
+  @JoinColumn({ name: 'event' })
+  @Index('event_feedback_event_index')
+  event: EventModel;
+
+  @ManyToOne((type) => UserModel, (user) => user.eventFeedback, { nullable: false })
+  @JoinColumn({ name: 'user' })
+  @Index('event_feedback_user_index')
+  user: UserModel;
+
+  @Column()
```

Comment:
it definitely doesn't need to be limited to 255 chars but I think there should be some character limit (frontend can decide that) since event feedback is generally meant to be short (at least that's what it looks like from the designs)

Reply:
update on this - custom event feedback can be as long as needed so no limit is necessary